lib/Db/Table.php

```diff
@@ -101,10 +102,9 @@ public function jsonSerialize(): array {
 	}
 
 	/**
-	 * @psalm-suppress MismatchingDocblockReturnType
-	 * @return array{read: bool, create: bool, update: bool, delete: bool, manage: bool}|null
+	 * @return ?Permissions
```

Comment:
No need as we have the type hint already.

Reply:
and removed 2, 3 others of the same